### PR TITLE
Documentation for AUP features

### LINKED ISSUE #116: Add AUP features included in v1.9.0
In particular:
- VO members can resign the AUP at any time: PR https://github.com/indigo-iam/iam/pull/757
- allow to delete a signature and also sign on behalf of the user: PR https://github.com/indigo-iam/iam/pull/777
- in case AUP screenshots are shown in this documentation, replace them with the ones displaying how much time is left until expiration (PR https://github.com/indigo-iam/iam/pull/783) rather than when it was signed last time.

### PR DESCRIPTION
**Pull request by garaimanoj**

### CODE REVIEW

File: content/en/docs/tasks/administration/aup-management/_index.md [hunk 1]
```diff
@@ -82,3 +82,55 @@ at registration time:
...
+
+To confirm your choice click on _Request AUP signature_ button on the modal window:
+
+![request AUP signature modal](../images/request-aup-signature-4.png)
+
+On success you will get a confirmation message:
+
+![request AUP signature confirmation](../images/request-aup-signature-5.png)
+
+After this operation if the user login, the user need to sign the AUP again before accessing the service.
```
Comment (rmiccoli) [inline content/en/docs/tasks/administration/aup-management/_index.md]: I would rewrite it like this: After this operation, if the user logs in, they must sign the AUP again before accessing the service.

File: content/en/docs/tasks/administration/aup-management/_index.md [hunk 2]
```diff
@@ -82,3 +82,55 @@ at registration time:
 or at the first login after the AUP creation:
 
 <img src="../images/aup4.png" width="500">
+
+## Request AUP signature
+
+Administrator can request user to sign AUP
```
Comment (rmiccoli) [inline content/en/docs/tasks/administration/aup-management/_index.md]: Put a full stop at the end of the sentence

File: content/en/docs/tasks/administration/aup-management/_index.md [hunk 3]
```diff
@@ -82,3 +82,55 @@ at registration time:
...
+
+On success you will get a confirmation message:
+
+![request AUP signature confirmation](../images/request-aup-signature-5.png)
+
+After this operation if the user login, the user need to sign the AUP again before accessing the service.
+
+## Sign AUP on behalf of the user
+
+Administrator can sign AUP on behalf of the user
```
Comment (rmiccoli) [inline content/en/docs/tasks/administration/aup-management/_index.md]: Put a full stop at the end of the sentence

**Review by rmiccoli — APPROVED** [review]

File: content/en/docs/tasks/administration/aup-management/_index.md (hunk 1 above)
Comment (garaimanoj) [reply content/en/docs/tasks/administration/aup-management/_index.md]: Done

File: content/en/docs/tasks/administration/aup-management/_index.md (hunk 2 above)
Comment (garaimanoj) [reply content/en/docs/tasks/administration/aup-management/_index.md]: Done

File: content/en/docs/tasks/administration/aup-management/_index.md (hunk 3 above)
Comment (garaimanoj) [reply content/en/docs/tasks/administration/aup-management/_index.md]: Done

File: content/en/docs/tasks/user/ resign-AUP/_index.md
Comment (rmiccoli) [inline content/en/docs/tasks/user/ resign-AUP/_index.md:1]: Remove the space in the URL before 'resign-AUP'.

Comment (garaimanoj) [reply content/en/docs/tasks/user/ resign-AUP/_index.md:1]: Done

**Pull request merged**